pkgs/os-specific/linux/swapspace/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, autoconf, automake, utillinux, btrfs-progs }:
+
+stdenv.mkDerivation rec {
+  version = "1.16.1";
+  name = "swapspace-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/Tookmund/Swapspace.git";
+    rev = "refs/tags/v${version}";
+    sha256 = "1hbxaq05zdfhw5c8lypnl1qvh7n9ng0nawlizs9g6l77z8sn0ji8";
+    fetchSubmodules = false;
```

Comment:
Why do you explicitly set this to false here? Does it provide submodules, but they're wrong?

AFAIK, `fetchSubmodules` is the default anyways.